src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCosts = 40;
```

Comment:
Consider using a more descriptive variable name than `dayCosts`. For example, `dailyRentalCost`. 🤔